packages/flame_steering_behaviors/example/pubspec.yaml

```diff
@@ -0,0 +1,22 @@
+name: example
```

Comment:
```suggestion
name: flame_steering_behaviors_example
```

Reply:
Keeping it as `example` as that is what the other example was as well, can always be changed in a follow-up PR